x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -117,16 +120,17 @@ private void doStartDeployment(TrainedModelDeploymentTask task) {
                 processContext.startProcess();
                 processContext.loadModel(modelLoadedListener);
             },
-            e -> failTask(task, e)
+            e -> failTask(task,
+                String.format(Locale.ROOT, "[%s] search for task config failed with error [%s]", task.getModelId(), e))
```

Comment:
I think it would be useful to log the trace here and above. I am not sure if the trace is included in the `Exception#toString` method.